docs/source/spec.rst

```diff
@@ -35,25 +36,27 @@ units of work meant to be run by a single worker. Example:
 
 We represent a task as a tuple such that the *first element is a callable
 function* (like ``add``), and the succeeding elements are *arguments* for that
-function.
+function. An *argument* may be any valid **computation**.
 
-An **argument** may be one of the following:
+A **computation** may be one of the following:
 
-1.  Any key present in the dask like ``'x'``
+1.  Any **key** present in the dask graph like ``'x'``
 2.  Any other value like ``1``, to be interpreted literally
-3.  Other tasks like ``(inc, 'x')``
-4.  List of arguments, like ``[1, 'x', (inc, 'x')]``
+3.  A **task** like ``(inc, 'x')`` (see below)
+4.  A list of **computations**, like ``[1, 'x', (inc, 'x')]``
 
-So all of the following are valid tasks:
+So all of the following are valid **computations**:
 
 .. code-block:: python
 
+   np.array([...])
```

Comment:
I'm somewhat in favor of punting and going with computation for now.  We can change this in the future if we come up with cleaner ideas.  I'd like to get this merged.